modules/ossm-multitenant.adoc

```diff
@@ -0,0 +1,41 @@
+////
+Module included in the following assemblies:
+-ossm-vs-community.adoc
```

Comment:
It may not have updated yet when you checked.  I discovered that there can be a bit of a time lag for the preview to update sometimes. It's updated now.  This module is part of the architecture Assembly, the heading is Differences between Istio and Service Mesh.

Reply:
@JStickler I still don't see it in the preview in either https://bz_1807736--ocpdocs.netlify.app/openshift-enterprise/latest/service_mesh/v1x/ossm-architecture.html or https://bz_1807736--ocpdocs.netlify.app/openshift-enterprise/latest/service_mesh/v1x/ossm-architecture.html. I see that the module was added to the v2 assembly. 

Is that where it was supposed to go? Is this for v1 or v2?

Reply:
Ah, sorry, I misread.  That module (without a version number) is going to be for version 2.0 and isn't published.   I need to open a JIRA and get the Maistra team to start thinking about what we need to document for version 2.0, because I'm sure there are going to be a different set of differences.

Reply:
I think it's a bit of a problem that it's not clear. I figured it out, but I see this causing issues down the line for teams that link to our docs. I'd like to hold off on merging this PR until we've had a chance to talk about our 2.0 and 2.1 naming strategy as a team.

Reply:
Added the multitenancy module to the v1x assembly.